src/Datadog.Trace/Tracer.cs

```diff
@@ -664,16 +671,10 @@ private static IDogStatsd CreateDogStatsdClient(TracerSettings settings, string
                                        $"lang_interpreter:{FrameworkDescription.Instance.Name}",
                                        $"lang_version:{FrameworkDescription.Instance.ProductVersion}",
                                        $"tracer_version:{TracerConstants.AssemblyVersion}",
-                                       $"service:{serviceName}"
+                                       $"service:{serviceName}",
+                                       $"{Tags.RuntimeId}:{RuntimeId}"
                                    };
 
-                var containerId = ContainerMetadata.GetContainerId();
-
-                if (containerId != null)
-                {
-                    constantTags.Add($"runtime-id:{containerId}");
```

Comment:
Should we keep this? (but as `container-id:{containerId}`

Reply:
I don't think so. I'm not aware of anything that will use it

Reply:
You're right. I thought you were removing this: https://github.com/DataDog/dd-trace-dotnet/blob/b68cdfef714b378cd61ced1fa2894e0e215d90a9/src/Datadog.Trace/Agent/Api.cs#L76